hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -1215,17 +1228,6 @@
       { instance: 'spi_device',      port: 'sd[1]',          connection: 'direct', pad: 'SPI_DEV_D1'   , desc: ''},
       { instance: 'spi_device',      port: 'sd[2]',          connection: 'direct', pad: 'SPI_DEV_D2'   , desc: ''},
       { instance: 'spi_device',      port: 'sd[3]',          connection: 'direct', pad: 'SPI_DEV_D3'   , desc: ''},
-      // USBDEV
```

Comment:
great!

Reply:
Indeed, this looks much better now - thanks for doing the cleanup work!